Sources/MapboxCoreNavigation/CoreConstants.swift

```diff
@@ -193,7 +193,7 @@ public extension Notification.Name {
      
      The user info dictionary contains the key `RouteController.NotificationUserInfoKey.roadNameKey`.
      */
-    static let routeControllerRoadName: Notification.Name = .init(rawValue: "RouteControllerRoadName")
+    static let currentRoadNameDidChange: Notification.Name = .init(rawValue: "CurrentRoadNameDidChange")
```

Comment:
Thanks, perhaps the documentation comment could also be updated to more explicitly refer to the road name changing.

Reply:
Thanks for the suggestion, I added this part in the changeLog instead, the documentation would be aligned with other `Notification.Name`.